client/jetpack-connect/controller.js

```diff
@@ -107,6 +108,23 @@ export function saveQueryObject( context, next ) {
 	next();
 }
 
+export function maybeOnboard( context, next ) {
```

Comment:
Maybe destructure: `const { query } = context;`, since youre using `context.query` properties so much in this function.

Reply:
Done in a04bf28f35, destructured directly in the list of arguments.